docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,715 @@
+        plt.legend()
+
+Now let's compare the different methods for a fixed number of tracks of equal length::
+
+    for method, estimates in results.items():
+        plot_accuracy(snrs, estimates, variances[method], label=method);
+
+    plt.ylim([-2, 3])
+
+.. image:: figures/long_track_estimates.png
+
+In this plot, we see the performance of the different methods.
+The shaded area indicates the area encapsulated by the mean ± standard deviation.
+It is clear from this plot that for SNR > 1 all of the methods perform equally well.
+We also see that the uncertainty estimates (indicated with the solid vertical lines) are pretty accurate on average.
+For lower SNRs, the precision quickly drops for CVE, while there is some bias for OLS.
+
+Ensemble estimates
+------------------
+
+What if the tracks are very short?
+We can simulate this scenario as well::
+
+    def simulate_tracks(snr):
+        """Function used to simulate tracks"""
+        params = snr_to_diffusion_parameters(snr)
+        tracks = lk.simulation.simulate_diffusive_tracks(**params, steps=8, num_tracks=500)
+        return tracks, params["diffusion_constant"]
+
+    snrs = 10**np.arange(-1, 1.1, 0.125)
+    results, _ = test_estimation_methods(simulate_tracks, snrs, methods=methods)
+    for method, estimates in results.items():
+        plot_accuracy(snrs, estimates, label=method);
+
+    plt.ylim([-5, 6])
+
+.. image:: figures/short_track_estimates.png
+
+In this case, getting precise per track estimates at low SNR is unrealistic (note the vertical axis range), since all the estimators perform poorly at low SNRs when there are only few points in the track.
+
+However, it might be possible to still get a single good estimate for an ensemble of tracks (analyzing multiple tracks at once).
+With ensemble analysis we assume that the diffusion for the individual tracks is the same.
+How to best aggregate multiple tracks to obtain such an estimate depends depends on the method of choice.
+
+Covariance-Based Estimator
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Aggregating results for the CVE is straightforward, since those can just be obtained by performing a weighted average of the per-track results.
+Here the weights are chosen to be the number of points contributing to each track.
+This way, longer tracks contribute more to the estimate than short tracks which is beneficial for both accuracy and precision.
+The weighted average is computed as :cite:`vestergaard2014optimal`:
+
+.. math::
+
+    \overline{\hat{D}} = \frac{\sum_{m=1}^M N_m \hat{D}_m}{\sum_{m=1}^M N_m}
+
+Here :math:`M` is the number of tracks, :math:`D_m` corresponds to the diffusion constant of track :math:`m` while :math:`N_m` corresponds to the number of data points contributing to its estimate.
+The associated variance of the weighted mean is given by:
+
+.. math::
+
+    \overline{\hat{D}} = \frac{\sum_{m=1}^M N_m \left(\hat{D}_m - \overline{\hat{D}}\right)^2}{\left(M - 1\right) \sum_{m=1}^M N_m}
```

Comment:
this has the same variable symbol as the mean above `\overline{\hat{D}}`

Reply:
Good catch! Changed it to 
```
    \mathrm{var}\left(\overline{\hat{D}}\right) = \frac{\sum_{m=1}^M N_m \left(\hat{D}_m - \overline{\hat{D}}\right)^2}{\left(M - 1\right) \sum_{m=1}^M N_m}
```